script/e2e-test.sh

```diff
@@ -19,6 +29,23 @@ ADDITIONAL_CLUSTER_IP=${7:-"172.18.0.3"}
 KAPP_CONTROLLER_VERSION=${8:-"v0.41.2"}
 CHARTMUSEUM_VERSION=${9:-"3.9.0"}
 IMG_PREFIX=${IMG_PREFIX:-"kubeapps/"}
+TESTS_GROUP=${TESTS_GROUP:-"${ALL_TESTS}"}
+
+group_is_supported=false
+for group in "${SUPPORTED_TESTS_GROUPS[@]}"; do
+  if [[ "${TESTS_GROUP}" == "${group}" ]]; then
+    group_is_supported=true
+    break
+  fi
+done
```

Comment:
We should be safe to use something like https://stackoverflow.com/a/15394738 rather than looping, shouldn't we?

Reply:
Yeah, I think that way the code would be more concise. I didn't know about that solution for searching items in an array, thanks for sharing!